src/rgw/setup/rgw_service

```diff
@@ -37,9 +37,13 @@ while [ $# -gt 0 ];  do
     shift 1
 done
 
+log_dir="$(dirname "${log_file}")"
+motr_log_path="$log_dir/m0_client"
```

Comment:
instead of 'm0_client' pls use name as 'motr_trace_files'

Reply:
Done